pkgs/by-name/ra/raycast/package.nix

```diff
@@ -40,14 +56,26 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   passthru.updateScript = lib.getExe (writeShellApplication {
     name = "raycast-update-script";
     runtimeInputs = [
+      cacert
       curl
       jq
-      common-updater-scripts
+      openssl
     ];
     text = ''
       url=$(curl --silent "https://releases.raycast.com/releases/latest?build=universal")
       version=$(echo "$url" | jq -r '.version')
-      update-source-version raycast "$version" --file=./pkgs/by-name/ra/raycast/package.nix
+
+      arm_url="https://releases.raycast.com/releases/$version/download?build=arm"
+      x86_url="https://releases.raycast.com/releases/$version/download?build=x86_64"
+
+      arm_hash="sha256-$(curl -sL "$arm_url" | openssl dgst -sha256 -binary | openssl base64)"
+      x86_hash="sha256-$(curl -sL "$x86_url" | openssl dgst -sha256 -binary | openssl base64)"
```

Comment:
could you tell me the command (how you would usually run the update script) you ran to test this? i've been relying on the ryan update bot too much these days

Reply:
I would usually run `APP=raycast; nix-shell --pure ./maintainers/scripts/update.nix --argstr package "$APP"`